js/client-mainmenu.js

```diff
@@ -1144,7 +1207,7 @@
 						if (count % bufBoundary == 0 && curBuf < 4) curBuf++;
 					}
 				} else {
-					bufs[curBuf] += '<li><button name="moreTeams" class="button">Show all teams</button></li>';
+					bufs[curBuf] += '<li><button name="moreTeamsPress" class="button">Show all teams</button></li>';
```

Comment:
I'm confused why this was renamed?

Reply:
Created instance with moreTeams, it was getting confused with this one in the call

Reply:
Could just change instance name...

Reply:
yea sorry, ihad to create this because i'm reinitializing the instance upon all the button calls, and didn't want the moreteams things to just close every time. but i'm changing the naming right now